pkgs/top-level/python-packages.nix

```diff
@@ -2084,6 +2084,26 @@ in modules // {
     };
   };
 
+  cheat = buildPythonPackage rec {
+    version = "2.1.26";
+    name = "cheat-${version}";
+
+    propagatedBuildInputs = with self; [ docopt pygments ];
+
+    src = pkgs.fetchFromGitHub {
```

Comment:
the package seems to be available on PyPI. For Python packages PyPI is the preferred source.